docs/conf.py

```diff
@@ -57,9 +58,13 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+# html_static_path = ["_static"]
```

Comment:
**question:** Does this need to be commented?

Reply:
AFIK this was originally left as a commented out piece of code in the conf.py template, but I'll go ahead and remove it.

Reply:
It's currently not commented on `main`, so unless the change is required, I think we can leave it as-is,